epsagon/modules/fastapi.py

```diff
@@ -0,0 +1,40 @@
+"""
+fastapi patcher module.
+"""
+
+from __future__ import absolute_import
+import wrapt
+from fastapi.routing import APIRoute
+from ..wrappers.fastapi import TracingAPIRoute
+from ..utils import is_lambda_env
+
+def _wrapper(wrapped, _instance, args, kwargs):
+    """
+    Adds TracingRoute into APIRouter (FastAPI).
+    :param wrapped: wrapt's wrapped
+    :param instance: wrapt's instance
+    :param args: wrapt's args
+    :param kwargs: wrapt's kwargs
+    """
+
+    # Skip on Lambda environment since it's not relevant and might be duplicate
+    if is_lambda_env():
+        return wrapped(*args, **kwargs)
+    route_class = kwargs.get('route_class', APIRoute)
+    if route_class != APIRoute:
+        # custom routes are not supported
```

Comment:
How about a debug print in that case - "custom fastapi route ignored", along with whatever data we have on the route?